app/services/submission_service.rb

```diff
@@ -311,5 +311,20 @@ def update_submission_info(artwork, submission)
       )
       submission.save!
     end
+
+    def add_user_to_submission(submission, gravity_user_id, access_token)
+      if gravity_user_id.present?
+        user = User.find_or_create_by(gravity_user_id: gravity_user_id)
+        submission.update!(user_id: user&.id)
+
+        # "send_new_receipt_email" is here for us to test the feature on staging
```

Comment:
**question:** I don't understand how this helps us test on Staging. Could you provide some more details, please?

Reply:
Oh! That's the name of the config that we're using like a feature flag. I'm sorry for making you explain that 🙏